newIDE/app/src/MainFrame/index.js

```diff
@@ -1983,10 +1983,12 @@ const MainFrame = (props: Props) => {
                 currentProject &&
                 hasAPreviousSaveForEditorTabsState(currentProject)
               ) {
-                openEditorTabsFromPersistedState(currentProject);
+                const openedEditorsCount = openEditorTabsFromPersistedState(
+                  currentProject
+                );
                 setIsLoadingProject(false);
                 setLoaderModalProgress(null, null);
-                openProjectManager(false);
+                openProjectManager(openedEditorsCount === 0);
```

Comment:
Should it also open the first scene?

Reply:
You're right, it will be consistent with the opening of a new project with multiple scenes